hicommonwealth/commonwealth
# [Launchpad Discovery]: Token discovery on explore page

## Link to Issue
Closes: https://github.com/hicommonwealth/commonwealth/issues/8803

## Description of Changes
- Add token discovery list on explore page
- Token pricing (market cap, market cap goal, current price, 24hr price change) are static atm and will be added in a followup ticket https://github.com/hicommonwealth/commonwealth/issues/9694

## "How We Fixed It"
N/A

## Test Plan
1. Enable `FLAG_TOKENIZED_COMMUNITY`
2. Visit `/communities`
3. Verify you see tokens list, if you have previously created any tokens, if not, verify you see an empty tokens placeholder, with an option to launch your own token.
4. Verify you can click on "see more" button to get more tokens (if total tokens in your local env is > 8)
5. Verify you see correct loading state per initial/subsequent loading of tokens
6. Verify that clicking on the token card redirects to its community
7. Verify cards are responsive across screen sizes
8. Verify you don't see the tokens list when `FLAG_TOKENIZED_COMMUNITY` is disabled

## Deployment Plan
N/A

## Other Considerations
N/A
